backend/src/rss.rs

```diff
@@ -114,6 +124,22 @@ async fn video_items(
     let query = format!("select {selection} from events where part_of = $1");
     let rows = db.query_raw(&query, dbargs![&series_oc_id]).await?;
 
+    fn map_tracks(tracks: &Vec<EventTrack>, doc: &mut ogrim::Document) -> () {
+        xml!(doc,
+            <media:group>
+                {|doc| for track in tracks {
+                    xml!(doc,
+                        <media:content
+                            url={track.uri}
+                            {..track.mimetype.clone().map(|t| ("type", t))}
```

Comment:
```suggestion
                            {..track.mimetype.as_ref().map(|t| ("type", t))}
```

This avoids cloning the `String`